### PR TITLE
Add more Chinese translations

### PR DESCRIPTION
**Pull request by ChenQuan**
## What do these changes do?

Added more Chinese translations.

## Related issue number
Nothing.

### CODE REVIEW

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/datasource.po [hunk 1]
```diff
@@ -28,6 +28,8 @@ msgid ""
 "creation routine <routines/creation>` and :doc:`random sampling "
 "<routines/random>`."
 msgstr ""
+"您可以像对象一样从Python数组创建mars tensor，就像Numpy一样，"
+"或者直接从Numpy数组创建。关于 :doc:`阵列创建例程<routines/creation>` 和 :doc:`随机抽样<routines/random>` 的更多细节。
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/datasource.po]: '阵列创建例程' -> '数组创建例程'

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/datasource.po [hunk 2]
```diff
@@ -39,31 +41,33 @@ msgstr ""
...
 msgid "Create sparse tensor"
-msgstr ""
+msgstr "创建稀疏张量"
 
 #: ../../source/tensor/datasource.rst:30
 msgid ""
 "Mars tensor can be sparse, unfortunately, only 2-D sparse tensors are "
 "supported for now, multi-dimensional tensor will be supported later soon."
 msgstr ""
+"不幸的是，Mars张量可能是稀疏的，现在只支持2-D稀疏张量，很快就会支持多维张量。"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/datasource.po]: The order could be adjusted, 'Mars张量可能是稀疏的，然而，现在只支持二维稀疏张量'.

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/datasource.po [hunk 3]
```diff
@@ -74,46 +78,57 @@ msgid ""
 "fact is that chunk's size may effect heavily on the performance of "
 "execution."
 msgstr ""
+"Mars 张量中，我们将张量分块处理。"
+" ``chunk_size`` 不是必须的，对于默认设置，块的字节占用将为128M。"
+"但是，用户可以以更灵活的方式指定每个块的大小，这可以适应数据规模。"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/datasource.po]: Also about the order, better to translate as "但是，用户可以根据数据规模以更灵活的方式指定每个块的大小“.

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/datasource.po [hunk 4]
```diff
@@ -74,46 +78,57 @@ msgid ""
...
 
 #: ../../source/tensor/datasource.rst:58
 msgid "Assume we have such a tensor with the data shown below."
 msgstr ""
+"假设我们有这样的张量，数据如下所示。"
 
 #: ../../source/tensor/datasource.rst:71
 msgid "We will show how different ``chunk_size=`` arguments will tile the tensor."
 msgstr ""
+"我们将展示不同的 ``chunk_size=`` 论证将如何平衡张量。"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/datasource.po]: Translation could be "我们将展示不同的 ``chunk_size=`` 将如何影响张量的分块".

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/eager-mode.po [hunk 5]
```diff
@@ -30,24 +30,28 @@ msgid ""
...
 "to debug."
 msgstr ""
+"Mars支持eager模式，使其易于开发和易于调试。"
 
 #: ../../source/tensor/eager-mode.rst:9
 msgid ""
 "Users can enable the eager mode by options, set options at the beginning "
 "of the program or console session."
 msgstr ""
+"用户可以通过选项启用热切模式，在程序开头或控制台会话中设置选项。"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/eager-mode.po]: 热切模式 -> eager 模式

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/eager-mode.po [hunk 6]
```diff
@@ -30,24 +30,28 @@ msgid ""
...
 "If eager mode is on, tensor will be executed immediately by default "
 "session once it is created."
 msgstr ""
+"如果打开了eager模式，则会在创建默认会话后立即执行tensor。"
 
 #: ../../source/tensor/eager-mode.rst:43
 msgid "Use ``fetch`` to obtain numpy value from a tensor:"
 msgstr ""
-
+"用于 ``fetch`` 从张量中获取numpy值："
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/eager-mode.po]: 用于 -> 使用

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/execution.po [hunk 7]
```diff
@@ -19,51 +19,57 @@ msgstr ""
...
 "``session.run`` methods."
 msgstr ""
+"除非用户调用 ``execute`` 或使用 ``session.run`` 方法，否则不会执行Mars张量。"
 
 #: ../../source/tensor/execution.rst:6
 msgid ""
 "If no session is created explicitly, the ``execute`` will create a local "
 "session, and mark it as a default session."
 msgstr ""
+"如果未明确 ``execute`` 创建会话，则将创建本地会话，并将其标记为默认会话。"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/execution.po]: This translation could be "如果没有显式创建回话，``execute`` 方法会创建一个本地回话，并将其标记为默认会话".

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/execution.po [hunk 8]
```diff
@@ -19,51 +19,57 @@ msgstr ""
...
 "is provided, a local session will be generated."
 msgstr ""
+"用户可以通过 ``new_session`` 方法创建新会话，如果未提供参数，则将生成本地会话。"
 
 #: ../../source/tensor/execution.rst:21
 msgid ""
 "By calling ``as_default`` of a session, the session will be marked as the"
 " default session."
 msgstr ""
+"通过调用 ``as_default`` 会话，会话将被标记为默认会话。"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/execution.po]: Better to translate as "通过调用会话的 ``as_default``方法，该会话将被标记为默认会话“

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/execution.po [hunk 9]
```diff
@@ -19,51 +19,57 @@ msgstr ""
...
 
 #: ../../source/tensor/execution.rst:49
 msgid "Execute a tensor"
-msgstr ""
+msgstr "执行张量"
 
 #: ../../source/tensor/execution.rst:51
 msgid "For a single tensor, ``execute`` can be called."
 msgstr ""
+"对于单个张量，``execute`` 可以调用。"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/execution.po]: Translation could be "可以调用张量的 ``execute`` 方法来执行单个张量。"

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/execution.po [hunk 10]
```diff
@@ -19,51 +19,57 @@ msgstr ""
...
 
 #: ../../source/tensor/execution.rst:59
 msgid "Session can be specified by the argument ``session=``."
 msgstr ""
-
+"会话可以由参数指定 ``session=``。"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/execution.po]: Translation could be "会话可以由参数 ``session=`` 指定"。

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/index.po [hunk 11]
```diff
@@ -20,4 +20,4 @@ msgstr ""
 #: ../../source/tensor/index.rst:2
 msgid "Mars tensor - distributed tensor with NumPy-like API"
 msgstr ""
-
+"Mars tensor  - 与类似NumPy的API分布张量"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/index.po]: Translation could be "Mars tensor - 提供类似 Numpy API 的分布式张量库"

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/logic.po [hunk 12]
```diff
@@ -20,10 +20,12 @@ msgstr ""
 #: ../../source/tensor/logic.rst:2
 msgid "Logic Functions"
 msgstr ""
+"Logic函数"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/logic.po]: Just call "逻辑函数" is fine.

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/logic.po [hunk 13]
```diff
@@ -20,10 +20,12 @@ msgstr ""
...
 
 #: ../../source/tensor/logic.rst:5
 msgid "Truth value testing"
 msgstr ""
+"Truth value 测试"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/logic.po]: Can be translated as "真值测试".

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po [hunk 14]
```diff
@@ -29,20 +30,27 @@ msgid ""
 "directed graph. This lets us compute on tensors larger than memory and "
 "take advantage of the ability of multi-cores or distributed clusters."
 msgstr ""
+"Mars tensor是Numpy的对应物，``numpy.ndarray`` 它实现了Numpy ndarray接口的一个子集。"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po]: Mars tensor是Numpy的对应物，``numpy.ndarray`` -> Mars tensor 对应 Numpy ndarray

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po [hunk 15]
```diff
@@ -29,20 +30,27 @@ msgid ""
...
+"它将一个大张量拼贴成小块，并用有向图描述内部计算。这使我们可以在大于内存的张量上进行计算，"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po]: 它将一个大张量拼贴成小块，并用有向图描述内部计算 -> 它将一个大张量切分成很多小块，并用有向图描述块之间的计算。

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po [hunk 16]
```diff
@@ -29,20 +30,27 @@ msgid ""
...
 "Arithmetic and mathematics: ``+``, ``-``, ``*``, ``/``, ``exp``, ``log``,"
 " etc."
 msgstr ""
+"算术和数学：``+``, ``-``, ``*``, ``/``, ``exp``, ``log`` 等"
+
 
 #: ../../source/tensor/overview.rst:12
 msgid "Reduction along axes (``sum``, ``max``, ``argmax``, etc)."
 msgstr ""
+"沿轴线还原(``sum``, ``max``, ``argmax``, etc)"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po]: 沿轴线还原(``sum``, ``max``, ``argmax``, etc) -> 沿轴线聚合(``sum``, ``max``, ``argmax``, 等)

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po [hunk 17]
```diff
@@ -52,33 +60,47 @@ msgid ""
 "Mars does not only support create array/tensor on GPU, but also support "
 "create sparse tensor."
 msgstr ""
+"大多数的 `阵列创建程序 "
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po]: 阵列 -> 数组

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po [hunk 18]
```diff
@@ -52,33 +60,47 @@ msgid ""
...
+"<https://docs.scipy.org/doc/numpy/reference/routines.array-"
+"creation.html>`_ (``empty``, ``ones_like``, ``diag``, etc)。"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po]: etc -> 等

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po [hunk 19]
```diff
@@ -52,33 +60,47 @@ msgid ""
...
+"更重要的是，Mars不仅支持在GPU上创建数组/张量，还支持创建稀疏张量。"
+
 
 #: ../../source/tensor/overview.rst:16
 msgid ""
 "Most of the `array manipulation routines "
 "<https://docs.scipy.org/doc/numpy/reference/routines.array-"
 "manipulation.html>`_ (``reshape``, ``rollaxis``, ``concatenate``, etc.)"
 msgstr ""
+"大部分的 `阵列操作例程 "
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po]: 阵列 -> 数组

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po [hunk 20]
```diff
@@ -52,33 +60,47 @@ msgid ""
...
 
 #: ../../source/tensor/overview.rst:18
 msgid ""
 "`Basic indexing "
 "<https://docs.scipy.org/doc/numpy/reference/arrays.indexing.html>`_ "
 "(indexing by ints, slices, newaxes, and Ellipsis)"
 msgstr ""
+"`基础索引 "
+"<https://docs.scipy.org/doc/numpy/reference/arrays.indexing.html>`_ "
+"(通过整数，切片，newaxes和省略号进行索引)"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po]: newaxes -> 添加新维度

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po [hunk 21]
```diff
@@ -52,33 +60,47 @@ msgid ""
...
 
 #: ../../source/tensor/overview.rst:20
 msgid ""
 "Fancy indexing along single axis with lists or numpy arrays, e.g. x[[1, "
 "4, 8], :5]"
 msgstr ""
+"使用列表或numpy数组沿单轴绘制索引，例如x [[1,4,8]，：5]"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po]: 绘制索引 -> 的花式索引

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po [hunk 22]
```diff
@@ -87,6 +109,9 @@ msgid ""
 "including product (``dot``, ``matmul``, etc.) and decomposition "
 "(``cholesky``, ``svd``, etc.)."
 msgstr ""
+"`线性代数函数 "
+"<https://docs.scipy.org/doc/numpy/reference/routines.linalg.html>`_, "
+"包括乘积 (``dot``, ``matmul``, etc.)和分解"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po]: etc -> 等

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po [hunk 23]
```diff
@@ -95,25 +120,30 @@ msgid ""
 "community is sincerely welcomed. The main feature not implemented are "
 "listed below:"
 msgstr ""
-
+"然而，Mars还没有实现整个Numpy接口，无论是时间限制还是难度都是主要障碍。"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po]: 无论是时间限制还是难度都是主要障碍。 -> 时间限制和难度是主要原因

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po [hunk 24]
```diff
@@ -95,25 +120,30 @@ msgid ""
...
 msgid "Only small subset of ``np.linalg`` are implemented."
 msgstr ""
+"只实现了一小部分 ``np.linalg``"
 
 #: ../../source/tensor/overview.rst:31
 msgid ""
 "Operations like ``sort`` which is hard to execute in parallel are not "
 "implemented."
 msgstr ""
+"操作就像 ``sort`` 这是很难并行执行不落实。"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po]: This translation could be "像 ``sort`` 这些操作很难高效并行"

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po [hunk 25]
```diff
@@ -95,25 +120,30 @@ msgid ""
...
 
 #: ../../source/tensor/overview.rst:32
 msgid ""
 "Mars tensor doesn't implement interface like ``tolist`` and ``nditer`` "
 "etc, because the iteration or loops over a large tensor is very "
 "inefficient."
 msgstr ""
-
+"Mars张量没有实现类似的界面 ``tolist`` 和 ``nditer`` 等，"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po]: Mars张量没有实现类似的界面 ``tolist`` 和 ``nditer`` 等， -> Mars张量没有实现类似 ``tolist`` 和 ``nditer`` 等接口，

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/set.po [hunk 26]
```diff
@@ -20,10 +20,12 @@ msgstr ""
 #: ../../source/tensor/set.rst:2
 msgid "Set routines"
 msgstr ""
+"设置例程"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/set.po]: 设置 -> 集合

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/sorting.po [hunk 27]
```diff
@@ -104,6 +106,7 @@ msgstr ""
 #: ../../source/tensor/sorting.rst:24
 msgid "Counting"
 msgstr ""
+"计算"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/sorting.po]: 计算 -> 计数

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/sparse.po [hunk 28]
```diff
@@ -20,17 +20,20 @@ msgstr ""
...
 msgstr ""
+"稀疏张量"
 
 #: ../../source/tensor/sparse.rst:4
 msgid ""
 "Mars tensor supports sparse tensor, unfortunately, only 2-D sparse "
 "tensors are available for now. Multi-dimensional sparse tensors will be "
 "supported later."
 msgstr ""
+"不幸的是，Mars张量支持稀疏张量，现在只有2-D稀疏张量可用。稍后将支持多维稀疏张量。"
```
Comment (chineking) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/sparse.po]: Ditto about order, could be "Mars支持稀疏张量，但不幸的是，目前只支持二维稀疏张量，稍后将支持多维稀疏张量"

**Comment by ChenQuan** [comment]
> Really thanks for your contribution, I left a few comments.

It has been revised.

**Comment by ChenQuan** [comment]
@chineking Thank you very much for your patience.

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/datasource.po [hunk 29]
```diff
@@ -28,6 +28,8 @@ msgid ""
 "creation routine <routines/creation>` and :doc:`random sampling "
 "<routines/random>`."
 msgstr ""
+"您可以像对象一样从Python数组创建mars tensor，就像Numpy一样，"
```
Comment (wjsi) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/datasource.po]: ``array like object`` is a whole phrase and can not be torn apart in translation.

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/index.po (hunk 11 above)
Comment (wjsi) [reply docs/source/locale/zh_CN/LC_MESSAGES/tensor/index.po]: This is not typical Chinese sentence.

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/ufunc.po [hunk 30]
```diff
@@ -20,39 +20,48 @@ msgstr ""
...
 msgid "Casting rules"
 msgstr ""
+"类型转换规则"
 
 #: ../../source/tensor/ufunc.rst:11
 msgid ""
 "Mars tensor's ufunc currently does not support methods like ``reduce``, "
 "``accumulate``, ``reduceat``, ``outer``, and ``at``."
 msgstr ""
+"Mars张量ufunc目前不支持包括方法有 ``accumulate``, ``reduceat``, ``outer``, 和 ``at``。"
```
Comment (wjsi) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/ufunc.po]: Comma and ``和`` should not be used simultaneously.

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/ufunc.po [hunk 31]
```diff
@@ -20,39 +20,48 @@ msgstr ""
...
 msgstr ""
+"通用函数(ufunc)"
 
 #: ../../source/tensor/ufunc.rst:4
 msgid ""
 "Mars tensor provides universal functions(a.k.a ufuncs) to support various"
 " elementwise operations. Mars tensor's ufunc supports following features "
 "of Numpy's one:"
 msgstr ""
+"Mars tensor提供通用函数(a.k.a ufuncs)来支持各种元素操作。"
```
Comment (wjsi) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/ufunc.po]: ``a.k.a.`` needs to be translated.

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/execution.po [hunk 32]
```diff
@@ -19,51 +19,57 @@ msgstr ""
...
 "``session.run`` methods."
 msgstr ""
+"除非用户调用 ``execute`` 或使用 ``session.run`` 方法，否则不会执行Mars张量。"
 
 #: ../../source/tensor/execution.rst:6
 msgid ""
 "If no session is created explicitly, the ``execute`` will create a local "
 "session, and mark it as a default session."
 msgstr ""
+"如果没有显式创建回话， ``execute`` 方法会创建一个本地回话，并将其标记为默认会话。"
```
Comment (wjsi) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/execution.po]: 回话?

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/creation.po [hunk 33]
```diff
@@ -19,11 +19,11 @@ msgstr ""
 
 #: ../../source/tensor/creation.rst:2
 msgid "Tensor Creation Routines"
-msgstr ""
+msgstr "Tensor创建例程"
```
Comment (wjsi) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/creation.po]: I wonder if the translation ``例程`` for ``routines`` is appropriate, as we does not only provide examples but also documentations of APIs. This applies to all ``routines`` in the translations below.

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/fft.po [hunk 34]
```diff
@@ -20,10 +20,12 @@ msgstr ""
 #: ../../source/tensor/fft.rst:4
 msgid "Discrete Fourier Transform"
 msgstr ""
+"离散傅立叶变换"
 
 #: ../../source/tensor/fft.rst:8
 msgid "Standard FFTs"
 msgstr ""
+"标准FFTs"
```
Comment (wjsi) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/fft.po]: FFTs needs translation

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po [hunk 35]
```diff
@@ -52,33 +60,47 @@ msgid ""
...
+"<https://docs.scipy.org/doc/numpy/reference/routines.array-"
+"manipulation.html>`_ (``reshape``, ``rollaxis``, ``concatenate``, 等)"
 
 #: ../../source/tensor/overview.rst:18
 msgid ""
 "`Basic indexing "
 "<https://docs.scipy.org/doc/numpy/reference/arrays.indexing.html>`_ "
 "(indexing by ints, slices, newaxes, and Ellipsis)"
 msgstr ""
+"`基础索引 "
```
Comment (wjsi) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/overview.po]: 基础 -> 基本

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/set.po [hunk 36]
```diff
@@ -20,10 +20,12 @@ msgstr ""
 #: ../../source/tensor/set.rst:2
 msgid "Set routines"
 msgstr ""
+"集合例程"
 
 #: ../../source/tensor/set.rst:5
 msgid "Boolean operations"
 msgstr ""
+"Boolean运算"
```
Comment (wjsi) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/set.po]: ``Boolean`` can be translated.

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/sparse.po [hunk 37]
```diff
@@ -20,17 +20,20 @@ msgstr ""
...
 msgstr ""
+"稀疏张量"
 
 #: ../../source/tensor/sparse.rst:4
 msgid ""
 "Mars tensor supports sparse tensor, unfortunately, only 2-D sparse "
 "tensors are available for now. Multi-dimensional sparse tensors will be "
 "supported later."
 msgstr ""
+"Mars支持稀疏张量，但不幸的是，目前只支持二维稀疏张量，稍后将支持多维稀疏张量"
```
Comment (wjsi) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/sparse.po]: ``later`` is not ``soon``.

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/datasource.po [hunk 38]
```diff
@@ -74,46 +78,57 @@ msgid ""
...
 
 #: ../../source/tensor/datasource.rst:56
 msgid "Chunks Examples"
-msgstr ""
+msgstr "块的例子"
 
 #: ../../source/tensor/datasource.rst:58
 msgid "Assume we have such a tensor with the data shown below."
 msgstr ""
+"假设我们有这样的张量，数据如下所示。"
```
Comment (wjsi) [inline docs/source/locale/zh_CN/LC_MESSAGES/tensor/datasource.po]: "假设我们有包含如下数据的张量"

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/fft.po (hunk 34 above)
Comment (ChenQuan) [reply docs/source/locale/zh_CN/LC_MESSAGES/tensor/fft.po]: Translation into `快速傅里叶变换`？

File: docs/source/locale/zh_CN/LC_MESSAGES/tensor/datasource.po (hunk 29 above)
Comment (ChenQuan) [reply docs/source/locale/zh_CN/LC_MESSAGES/tensor/datasource.po]: I think it may be `您可以像Numpy一样从Python数组对象中创建mars张量`, are you?

**Comment by ChenQuan** [comment]
I have a question, similar to the picture below, should I continue to translate?
image

**Comment by ChenQuan** [comment]
> Chinese translation of numpy docs (https://www.numpy.org.cn/) may be helpful in translating terms like `ufuncs` and `routines`. What's more, terms should be consistent in the whole document.

Thank you, I will read it carefully.

**Comment by chineking** [comment]
The latest commit looks good to me, if CI passed, I will merge this PR into master branch.

Hope this is a start to get involved in Mars development for you, not just the translation. Thanks.

**Comment by chineking** [comment]
I am going to merge this, any questions further? @wjsi , if not, approve this.

**Comment by ChenQuan** [comment]
> The latest commit looks good to me, if CI passed, I will merge this PR into master branch.
> 
> Hope this is a start to get involved in Mars development for you, not just the translation. Thanks.

I am very happy to contribute and I will continue to do so. Try to fix the bug.

**Comment by ChenQuan** [comment]
> I have a question, similar to the picture below, should I continue to translate?
> image

@chineking @wjsi

**Comment by chineking** [comment]
> > I have a question, similar to the picture below, should I continue to translate?
> > image
> 
> @chineking @wjsi

You can open an issue if you have any further discussion.

**Comment by ChenQuan** [comment]
> ou can open an issue if you have any further discussion.

ok,I will do this.thanks

**Pull request merged**